ibis/formats/parser.py

```diff
@@ -0,0 +1,213 @@
+from __future__ import annotations
+
+import abc
+from typing import Mapping
+
+import sqlglot as sg
+from sqlglot.expressions import ColumnDef, DataType, DataTypeSize
+
+import ibis.common.exceptions as exc
+import ibis.expr.datatypes as dt
+from ibis.common.bases import Singleton, Slotted
+from ibis.common.collections import FrozenDict
+
+SQLGLOT_TYPE_TO_IBIS_TYPE = {
+    DataType.Type.BIGDECIMAL: dt.Decimal(76, 38),
```

Comment:
Can you alias `DataType.Type` before the mapping, e.g. `sgt = DataType.Type`?

Reply:
I'd rather leave it like this. While it's very repetitive, it's also one less level of indirection.

Reply:
Fair enough.